### PR TITLE
ds-rshiny upgrade and housekeeping

### LINKED ISSUE #563: ds-rshiny Dockerfile update
**Is your feature request related to a problem? Please describe.**
The `ds-rshiny` is still using `ubuntu:16.04` as the base image. We need to upgrade it and port to a better managed base image.

**Describe the solution you'd like**
To have a more recent OS base image (maybe UBI based) or to use one from the widely used https://github.com/rocker-org

**Describe alternatives you've considered**
To just upgrade the current ubuntu version.

**Additional context**
IMHO we should use base images that are not too much opinionated like maybe the rocker based ones are but just provide a really minimal baseline as we already do. Users will anyway tweak it a lot, it is R!

### PR DESCRIPTION
**Pull request by gerardcl**
I decided to focus on upgrading current offered base OS, so to update to Ubuntu 20.04.
I also did a first try with UBI8 and that will require a bit of more tweaks in terms of adding manually many dependencies required for having R in a proper shape. But it will be possible.

Fixes #563 

Tasks: 
- [x] Updated documentation in `docs/modules/...` directory
- [x] Ran tests in `<quickstarter>/testdata` directory

### CODE REVIEW

File: ds-rshiny/files/docker/Dockerfile [hunk 1]
```diff
@@ -35,25 +36,30 @@ RUN apt-get update && \
...
-RUN R -e 'packages = c("RJDBC", "shinythemes", "ggplot2", "reshape2"); \
+RUN R -e 'packages = c("shinythemes", "ggplot2", "reshape2"); \
     install.packages(packages, repos="http://cran.uni-muenster.de/"); \
     package.check <- lapply(packages, FUN = function(p) { if(!require(p, character.only = TRUE)) { stop("package not found!") } })'
 
 # Cleanup apt
-RUN apt-get clean && rm -rf /var/lib/apt/lists/*
+RUN apt-get autoremove --purge && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /var/cache/apt/archives
```
Comment (michaelsauter) [inline ds-rshiny/files/docker/Dockerfile:57]: This is a bit pointless actually. Given container images are layers, this does not actually make the image any smaller. I see it's already been like that before though. I'd either remove it, or, if possible, add it to the layer which runs `apt-get install`.

Comment (gerardcl) [reply ds-rshiny/files/docker/Dockerfile:57]: Makes sense! on the other side I see it is more to ensure that later on noone can make use of apt.

Will give it a try and come back! 👍

Comment (michaelsauter) [reply ds-rshiny/files/docker/Dockerfile:57]: Whether one can use `apt` is more a matter of permissions - if I'm not mistaken, even on OpenShift in building you can switch back to root and then do `update` and `install` all over again ;)

**Review by felipecruz91 — APPROVED** [review]

File: ds-rshiny/files/docker/Dockerfile (hunk 1 above)
Comment (gerardcl) [reply ds-rshiny/files/docker/Dockerfile]: ok, I did some refactor with success and it really looks much more better! thanks for taking a look @michaelsauter !! 🚀

**Review by michaelsauter — APPROVED** [review]

**Pull request merged**